cfpb/college-costs
# remove duplicate test; fix standalone js so we can see a saved worksheet

## Additions
- worksheet.json fixture for testing a saved worksheet
## Removals
- duplicate view test
## Changes
- fixed url in standalone js so we can see a saved worksheet
- updated comments in test_views.py with correct urls
## Review
- @ascott @niqjohnson @marteki 
## Testing (standalone or UnityBox)
- load the test worksheet with 

``` bash
./manage.py loaddata worksheet.json
```
- check this saved-worksheet url (you may need to refresh it once):

```
/paying-for-college/understanding-financial-aid-offers/#00470019-e077-4fc3-9dbb-4a595fe976e6
```

You should see a worksheet for ITT Technical Institute–Tallahassee:

<img width="913" alt="worksheet_test" src="https://cloud.githubusercontent.com/assets/515885/11668449/44b03f7c-9dc5-11e5-8ec6-4e44a47b11d1.png">
